firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -4,36 +4,83 @@
   0         1        2        3
   Step pin, Dir pin, MS1 pin, MS2 pin
 */
```

Comment:
Please update these comments describing pins explaining what each of the five elements of the motor array correspond to now that you've added the limit switch to the motor array (or simply remove the comments).